promotion-service/promotionservice/constants.py

```diff
@@ -0,0 +1,3 @@
+DATA_LIST_SIZE = 5
```

Comment:
Can we maintain at-least 10 random numbers here? As we are planning for 100 book data, better to have 10 promotions I guess.